sema4ai/vscode-client/src/pythonExtIntegration.ts

```diff
@@ -34,6 +37,16 @@ export function isPythonFile(fsPath: string): boolean {
     return fsPath.endsWith(".py");
 }
 
+function matchActionPackageToFile(actionPackages: LocalRobotMetadataInfo[], filePath: Uri): Uri | undefined {
+    const filteredPaths = actionPackages
+        .filter((actionPackage: LocalRobotMetadataInfo) => filePath.fsPath.includes(actionPackage.directory))
```

Comment:
I think you could use `find` instead, so you wouldn't need to bother with indexing later on ^^ and then only do
```
return filteredPackage ? Uri.file(filteredPackage.directory) : null;
```